storage/retrieval.go

```diff
@@ -133,69 +153,81 @@ func (b *Bucket) getLogChunks(ctx context.Context, buildID string) ([]LogChunkIn
 		}
 	}
 
-	if err := iter.Err(); err != nil {
```

Comment:
It looks like we stopped checking for err on iter here. Or did this move somewhere I'm missing?

Reply:
This got dropped.
I added it back.